src/Compilers/Core/Portable/MetadataReference/AssemblyIdentity.DisplayName.cs

```diff
@@ -723,6 +723,11 @@ private static bool IsWhiteSpace(char c)
 
         private static void EscapeName(StringBuilder result, string name)
         {
+            if (string.IsNullOrEmpty(name))
```

Comment:
Do we have any tests that verify such a display name can round try? For instance what happens when I call `AssemblyIdentity.TryParseDisplayName` on the result of `GetDisplayName` when the assembly name is an empty string?

Reply:
It seems like for now we will accept that this AssemblyIdentity produces a DisplayName that does not parse, and look into the issue more closely a little later.